sig-services/service-requirements/data-collection-permission.md

```diff
@@ -0,0 +1,22 @@
+---
+Title: Data Collection/Permission
+Description: social login
+Persona: service consumer, service provider/owner
+Phase: service-operational
+---
+
+This requirement states that a service should have a well defined social login description. Social login means the kind of information one gets from the user when one login to the serivce.
+
+It should also clearly state the reason behind getting the data priviledges in order to impersonate on their behalf.
+
+## Requirements
+
+- Service owner should be subscribed to the service.
```

Comment:
This guideline should focus on what content goes in a data collection and permissions doc. So points about how can we be clear to the service consumer about their data that we'll collect. Are we collecting their github id and name? Their Github org perms? Their browser session info? This guideline should say that if we are doing that then we should mention it in a doc.

Reply:
Thanks for the suggestions @Shreyanand . I have tried to update the doc. Lmk your feedback. :)